app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ Optional] I recommend you to use author instead of user.  we can optimize like four in this way. `belongs_to :author, Class_name: 'User'`